packages/core/src/claim/Claim.ts

```diff
@@ -189,7 +189,7 @@ export function verifyDisclosedAttributes(
 }
 
 /**
- * Checks whether the input meets all the required criteria of an [[IClaim]] object.
+ * Checks whether the input meets all the required criteria of an [[IClaim]] or [[IAssetClaim]] object.
```

Comment:
Guess this comment needs to be reverted as well

Reply:
Nice! https://github.com/KILTprotocol/sdk-js/pull/677/commits/589c8bc7d8f0fbdf240c2824a70adb34102dbfb0#diff-0e2cdf2fffcea87f7c76a48effc1b12d7758d3f545a5975fba1c0c71440fbee8R192